app/src/main/res/layout/fragment_search_location.xml

```diff
@@ -9,20 +8,46 @@
     <FrameLayout
         android:id="@+id/autocomplete_frag_container"
         android:layout_width="match_parent"
-        android:layout_height="0dp"
+        android:layout_height="wrap_content"
```

Comment:
@iamareebjamal here it results showing no results at all if set to wrap content it only works well with match parent.
Result after setting height to 250dp
![screenshot-1554173746539](https://user-images.githubusercontent.com/29139786/55373072-9db9ec00-5521-11e9-8aac-ef0e58df7a78.jpg)